lib/plugins/aurelia.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * This file is part of the Symfony Webpack Encore package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+'use strict';
+
+const PluginPriorities = require('./plugin-priorities');
+const loaderFeatures = require('../features');
+
+/**
+ * @param {Array} plugins
+ * @param {WebpackConfig} webpackConfig
+ * @return {void}
+ */
+module.exports = function(plugins, webpackConfig) {
+    if (!webpackConfig.useAurelia) return;
+
+    loaderFeatures.ensurePackagesExist('aurelia-webpack-plugin');
+
+    const AureliaPlugin = require.resolve('aurelia-webpack-plugin');
```

Comment:
Shouldn't that be `require('aurelia-webpack-plugin')` since a `require.resolve()` call would return a string and not a constructor?

Reply:
To be more precise it should probably be something like:
```js
const { AureliaPlugin } = require('aurelia-webpack-plugin');
```

or without the destructuring assignment:
```js
const AureliaPlugin = require('aurelia-webpack-plugin').AureliaPlugin;
```